### DISCUSSION TITLE
The arrows don't come out

### DISCUSSION THREAD

**Discussion by guminlianren** [bcakmakoglu/vue-flow#1579] [General]
const {
    nodes,
    edges,

  } = useVueFlow();

**Comment by guminlianren** [comment]
onConnect((connection) => {
    addEdges({
      ...connection,
      type: 'custom',
      selected: false,
      style: { stroke: userStore.getThemeColor },
      markerEnd: {
        type: 'arrow',
        width: 20,
        height: 30,
        color: userStore.getThemeColor,
      },
      markerStart: {
        type: '',
        width: 20,
        height: 30,
        color: userStore.getThemeColor,
      },
      // type: 'custom'
    });
  });      const edge = edges.value.find((v) => v.id === data.id);
      if (edge) {
        edge.style.stroke = data.stroke;
        edge.style.strokeWidth = data.stroke_width;
        edge.animated = data.animated === 'T' && true;
        const markerStart = {
          type: '',
          width: 20,
          height: 30,
        };
        const markerEnd = {
          type: '',
          width: 20,
          height: 30,
        };
        markerStart.type = '';
        markerEnd.type = '';
        if (data.marker === 1 || data.marker === 4) {
          markerStart.type = data.markerType;
          markerStart.color = data.stroke;
          edge.markerStart.type = data.markerType;
          edge.markerStart.color = data.stroke;
        }
        if (data.marker === 2 || data.marker === 4) {
          markerEnd.type = data.markerType;
          markerEnd.color = data.stroke;
          edge.markerEnd.type = data.markerType;
          edge.markerEnd.color = data.stroke;
        }
        VueFlowRef.value.updateEdgeData(edge.id, () => ({ markerEnd, markerStart }));
      }   I started with a close arrow, and then I want to add a start arrow. There's no effect on the back, it's already on the edges.

**Comment by bcakmakoglu** [comment]
There are some standards you need to uphold for questions:
* *What* are you trying to achieve.
* What have you tried 
* What hasn't worked

This post is completely unreadable, you didn't even bother writing more than a single sentence.
*Please* stop spamming with these sort of questions, it's annoying and shows very little respect for my time since you don't even bother formulating a proper question.
